src/rp2_common/pico_runtime/CMakeLists.txt

```diff
@@ -65,7 +65,17 @@ endif()
 # FLOAT                 - support for single-precision floating point
 # DOUBLE                - support for double-precision floating point
 # FPGA_CHECK            - checks for FPGA which allows Raspberry Pi to run your binary on FPGA
+# PANIC                 - default panic impl which brings in stdio
+# AUTO_INIT_MUTEX       - auto init mutexes; without this you get no printf mutex either               -
```

Comment:
Is it worth mentioning/documenting "ALL" in this comment?

Reply:
probably; it needs better documentation anyway

Reply:
i'll add more documentation in a separate PR